### ISSUE TITLE
Request for Improved Documentation and Use Case Examples

### ISSUE THREAD

**Issue by halbodb** [seanlowe/obsidian-timelines#101]
I am currently studying ancient history and use timelines daily in my research. I believe this plugin could solve many of my challenges. However, I’ve encountered some difficulties with the plugin, particularly due to the lack of real-world samples and clarity in the documentation.

Could you provide a clear use case example or consider updating your YouTube video to offer more accessible guidance for users like myself?

Your help would be greatly appreciated!

Thanks!

**Comment by flow7** [comment]
Hey there, maybe just to add to this - I also found it really really hard to consistently have my events show up in the timeline.
(Strongly prefer the Properties feature for adding/editing events, rather than html) 

Suggestion: can you make a vault available, with a couple of sample events, for us to go through and learn from?
thanks heaps - I love the general implementation!

**Comment by halbodb** [comment]
Yes, I agree, please make a demo vault available for us to understand how to use this great Obsidian plug-in. (This will be the one ring to rule them all! ~ J.R.R. Tolkien) 💍
